rules/integrations/aws_bedrock/aws_bedrock_high_confidence_misconduct_blocks_detected.toml

```diff
@@ -79,17 +79,43 @@ timestamp_override = "event.ingested"
 type = "esql"
 
 query = '''
-from logs-aws_bedrock.invocation-*
+FROM logs-aws_bedrock.invocation-*
+
+// Expand multi-value fields
 | MV_EXPAND gen_ai.compliance.violation_code
 | MV_EXPAND gen_ai.policy.confidence
 | MV_EXPAND gen_ai.policy.name
-| where gen_ai.policy.action == "BLOCKED" and gen_ai.policy.name == "content_policy" and gen_ai.policy.confidence LIKE "HIGH" and gen_ai.compliance.violation_code IN ("HATE", "MISCONDUCT", "SEXUAL", "INSULTS", "PROMPT_ATTACK", "VIOLENCE")
-| keep user.id, gen_ai.compliance.violation_code
-| stats block_count_per_violation = count() by user.id, gen_ai.compliance.violation_code
-| SORT block_count_per_violation DESC
-| keep user.id, gen_ai.compliance.violation_code, block_count_per_violation
-| STATS violation_count = SUM(block_count_per_violation) by user.id
-| WHERE violation_count > 5
-| SORT violation_count DESC
+
+// Filter for high-confidence content policy blocks with targeted violations
+| where
+  gen_ai.policy.action == "BLOCKED"
+  and gen_ai.policy.name == "content_policy"
+  and gen_ai.policy.confidence LIKE "HIGH"
+  and gen_ai.compliance.violation_code IN ("HATE", "MISCONDUCT", "SEXUAL", "INSULTS", "PROMPT_ATTACK", "VIOLENCE")
+
+// Keep ECS + compliance fields
+| keep
+  user.id,
+  gen_ai.compliance.violation_code
+
+// Count blocked violations per user per violation type
+| stats
+    Esql.ml_policy_blocked_violation_count = COUNT()
+  by
+    user.id,
+    gen_ai.compliance.violation_code
+
+// Aggregate all violation types per user
+| stats
+    Esql.ml_policy_blocked_violation_total_count = SUM(Esql.ml_policy_blocked_violation_count)
+  by
```

Comment:
nit: I think single line is easier to read, but im assuming its just really long now.